R/data.R

```diff
@@ -0,0 +1,65 @@
+#' Forecast outputs
+#'
+#' Example forecast data that represents model outputs from a hub (in this case, forecast data
+#' represents three influenza-related targets (wk inc flu hosp, wk flu hops rate category,
+#' and wk flu hosp tate) for two reference dates in 2022.
+#'
+#' @format ## `forecast_outputs`
+#' A data frame with 5,424 rows and 9 columns:
+#' \describe{
+#'   \item{location}{FIPS code identifying a location}
+#'   \item{reference_date}{the starting point of the forecast in yyyy-mm-dd format}
+#'   \item{horizon}{number of units ahead being forecasted (weeks, in this case)}
+#'   \item{target_end_date}{the date of occurrence of the outcome of interest in yyyy-mm-dd format;
+#'          this can be calculated directly from the `reference_date` and `horizon`
+#'          as follows: `target_end_date = reference_date + 7*horizon`}
+#'   \item{target}{a unique identifier for the target}
+#'   \item{output_type}{the type of representation of the prediction}
+#'   \item{output_type_id}{more identifying information specific to the output type;
+#'        output_type_id is not relevant for every kind of output_type (for example,
+#'        hubs will not expect output_type_id values when the output_type is mean or median}
+#'   \item{value}{the model’s prediction}
+#'   \item{model_id}{the name of the model}
```

Comment:
@elray1 question about `model_id`: is that a column we'd expect to see in a hub's model output data? I thought we derived it from the filename.

Reply:
It is true that when the data are sitting in a hub, the `model_id` is encoded in the file name.  But when we `collect` the data into a data frame in a working R (or in the future, python) session, the `model_id` is added into the data.  And the intent of this example object is to represent what a user might get after running `collect_hub()`.  (Maybe we should say that in this documentation.)

Reply:
Ah, makes sense--thank you for that clarification. Just pushed a commit with that note.